public/index.html

```diff
@@ -18,6 +18,8 @@
     <meta property="og:title" content="Join The Mastodon Project" />
     <meta property="og:description" content="Giving social networking back to you. With thousands of interconnected communities to choose from, and the tools to make your own, Mastodon is the world's largest free, open-source, decentralized micro-blogging network." />
     <meta property="og:type" content="website" />
+    <meta property="og:image" content="https://joinmastodon.org/mastodon-logo.png" />
+    <meta property="og:url" content="https://joinmastodon.org/" />
```

Comment:
We need full URL here due to https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url